tools/build_docs.sh

```diff
@@ -29,23 +29,31 @@ s3cmd --dump-config > s3.cfg
 # We are using docker because a library's documentation links to other
 # library documentation, and we want to guarantee a clean system.
 if [[ $1 == 'acropolis' || $1 == 'Acropolis' ]]; then
-  echo "Uploading documentation for Acropolis"
+  echo ::group::Acropolis
+  echo -e "\e[46m\e[90mUploading documentation for Acropolis\e[0m\e[39m"
   docker build -t ign-acropolis-docs -f Dockerfile.acropolis --build-arg IGN_VERSION_PASSWORD --build-arg IGN_VERSION_DATE=`date -Iseconds` --no-cache .
+  echo ::endgroup::
 fi
 
 if [[ $1 == 'blueprint' || $1 == 'Blueprint' ]]; then
-  echo "Uploading documentation for Blueprint"
+  echo ::group::Blueprint
+  echo -e "\e[46m\e[90mUploading documentation for Blueprint\e[0m\e[39m"
   docker build -t ign-blueprint-docs -f Dockerfile.blueprint --build-arg IGN_VERSION_PASSWORD --build-arg IGN_VERSION_DATE=`date -Iseconds` --no-cache .
+  echo ::endgroup::
```

Comment:
Should we make the same changes to `Dockerfile.blueprint` that were made to `Dockerfile.citadel` and `Dockerfile.dome`? Or does this not matter since blueprint is now EOL?

Reply:
I didn't bother updating the EOL ones, they just exist as historical reference and shouldn't be used.